suites/message/transfer.go

```diff
@@ -122,11 +122,12 @@ func MessageTest_ValueTransferSimple(t *testing.T, factories state.Factories) {
 			// create a message to transfer funds from `to` to `from` for amount `transferAmnt` and apply it to the state tree
 			// assert the actor balances changed as expected, the receiver balance should not change if transfer fails
 			if tc.code.IsSuccess() {
-				td.AssertBalance(tc.sender, big_spec.Sub(big_spec.Sub(tc.senderBal, tc.transferAmnt), result.Receipt.GasUsed.Big()))
+				td.AssertActor(tc.sender, tc.senderBal, result.Receipt, 1)
```

Comment:
What about the subtraction of `transferAmnt` from the sender's balance?

Reply:
That is subsumed into the `AssertActor` method.

Reply:
I'm really confused, how can it possibly know the right amount is `transferAmnt`?

Even if it's right, it's too confusing to be a good pattern.

Reply:
Ohhh, it's getting it from the "receipt" which includes the whole message. Way too confusing, I think we should not include that value in the receipt or result at all, so that tests must spell it out.